File: C++/MathematicalPrograms/PositiveOrNegativeChecker.cpp
//Program to check whether a number is Postitive or Negative
#include <iostream>
#include <string>

using namespace std;

int main() {
	int num;
	cout << "Enter a number";
	cin >> num;
	if (num > 0) {
		cout << "Your number " << num << " is positive";
	}
	else {
		cout << "Your number " << num << " is negative";
	}
}

File: Java/MathematicalPrograms/PositiveOrNegativeChecker.java
//Program to check whether a number is Postitive or Negative
import java.util.Scanner;
public class PositiveOrNegativeChecker {
	public static void main(String[] args) {
		 Scanner myObj = new Scanner(System.in);
		 System.out.println("Enter a number");
		 int num = myObj.nextInt();
		 if (num > 0) {
			 System.out.println("Your number " + num + " is positive");
		 }
		 else {
			 System.out.println("Your number " + num + " is negative");
		 }
	}
}
		 